### DISCUSSION TITLE
Skywalking installation as newbie

### DISCUSSION THREAD

**Discussion by silverraine** [apache/skywalking#10120] [Q&A]
Hello. I'm completely new to Skywalking and want to test it on my local. Is the showcase meant to be installed inside a pod or just run it from my shell? Been trying to find some tutorial videos on setup from ground up but cannot find them so I opened this discussion. 

Additionally I'm getting this error message when running make deploy.kubernetes

`validatingwebhookconfiguration.admissionregistration.k8s.io/cert-manager-webhook configured
process_begin: CreateProcess(NULL, sh ../../../scripts/wait-cert-manager-ready.sh, ...) failed.
make (e=2): The system cannot find the file specified.
make[1]: *** [features.mk:111: install-cert-manager] Error 2`

Thanks.

**Comment by wu-sheng** [comment]
@kezhenxu94 Do you know anything about this?
